js/components/draw/main.js

```diff
@@ -1,10 +1,9 @@
 import React from "react"
-import ReactDOM from "react-dom"
 import { Graph } from "../graph/Graph"
+import { createRoot } from 'react-dom/client';
```

Comment:
this import shouldn't have a semicolon, and should be on line 2 (replacing the old import of `react-dom`)